src/main/kotlin/no/nav/amt/deltaker/bff/deltaker/Deltaker.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.amt.deltaker.bff.deltaker
+
+import no.nav.amt.deltaker.bff.deltakerliste.Mal
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class Deltaker(
+    val id: UUID,
+    val personident: String,
+    val deltakerlisteId: UUID,
+    val startdato: LocalDate?,
+    val sluttdato: LocalDate?,
+    val dagerPerUke: Float?,
+    val deltakelsesprosent: Float?,
+    val bakgrunnsinformasjon: String?,
+    val mal: List<Mal>,
+    val status: DeltakerStatus,
+    val sistEndretAv: String?,
```

Comment:
Denne kan kanskje være påkrevd? Håper ikke vi endrer deltakeren uten å vite hvem som gjorde endringen 😇

Reply:
Ja den kan det. Det er bare at vi ikke har det på tidligere deltakere, men det er kanskje verdt å ha noe som identifiserer en systembruker eller noe.

Som fikk meg til å tenke, skal man skille på endringer gjort automatisk av system og manuelt av veileder? `modified_at` kan jo bli endret etter meldinger på kafka og lignende. 

Jeg synes selv at feltet er litt meningsløst slik det er nå, for nå vil vi overskrive hvem som gjorde forrige endring, men vi snakket jo om å vise hvem som gjorde siste endring i frontend så da er det vel greit å ha... Hvis dette skal vises i frontend blir det kanskje litt rart å vise `SYSTEMBRUKER` i de tilfellene 🤔